Documentation/dev-guide/interacting_v3.md

```diff
@@ -116,6 +141,28 @@ bar
 $ etcdctl get --rev=1 foo foo9 # access the versions of keys at revision 1
 ```
 
+## Read keys which are greater than or equal to the byte value of the specified key
+
+Applications may want to read all keys whose by value is greater than or equal to the specified key.
```

Comment:
all keys which are greater than or equal to the specified key when compared by bytes.